### DISCUSSION TITLE
不是哥们，这个darc为什么一直给我发别人的留言，这又不是我的项目?

### DISCUSSION THREAD

**Discussion by mianbaokuangfen** [Project-DARC/DARC#119] [Q&A]
如何防止darc发送无关邮件

**Comment by 1cryptoworld1** [comment]
你自己关注的，要设置一下才行

**Comment by defialways** [comment]
怎么设置啊
Kelvin
***@***.***
---- 回复的原邮件 ----
***@***.***>发送日期2024年05月10日 00:10 ***@***.***> ***@***.***>主题Re: [Project-DARC/DARC] 不是哥们，这个darc为什么一直给我发别人的留言，这又不是我的项目? (Discussion #119)
你自己关注的，要设置一下才行

**Comment by Betago-GitHub** [comment]
watch 时注意选择一下选项，不然所有信息更新都会收到
